watertap/flowsheets/full_water_resource_recovery_facility/test/test_BSM2_P_extension.py

```diff
@@ -334,13 +318,30 @@ def test_numerical_issues(self, system_frame):
 
     @pytest.mark.solver
     @pytest.mark.component
-    def test_condition_number(self, system_frame):
+    @linux_platform_only
+    def test_condition_number_on_linux(self, system_frame):
```

Comment:
Seems like this was the "fix" for the condition numbers not matching between Windows and Linux. I think this is fine as we've done similar things in other scaling/condition number tests already